aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,52 @@
+import { Example } from '../../layouts';
+import { PageContentExample, ResponsiveContentLayoutExample } from '../../examples';
+
+**Page Component** is the main container that is responsible for how the Page Content and Page Header are 
+presented. 
+
+<Example 
+  showResponsiveControls={['fullScreen']}
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.57 }}
+>
+  <ResponsiveContentLayoutExample />
+</Example>
+
+## Guidance
+Page takes `kind` as a prop which will determine the type of page layout. 
+There are 3 kinds of content width options depending on the use case. 
+1. [Wide (Default)](https://design-system.hpe.design/templates/page-layouts#wide-default)
```

Comment:
Might be good to mention the max width of the content is 1536px for `kind=wide`

Reply:
I can add that good thinking